recipes/flask-mail/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{%set name = "flask-mail" %}
+{%set camelName = "Flask-Mail" %}
```

Comment:
My impression was that we generally wanted package names to be lower-case, not camel case or mixed case. If that's not so, then sure, I can change it.